rules/source-control-tools/rule.md

```diff
@@ -0,0 +1,143 @@
+---
+type: rule
+title: Do you know some Source Control tools ?
+uri: source-control-tools
+authors:
+  - title: Gilles Pothieu
+    url: https://ssw.com.au/people/gilles-pothieu/
+created: 2024-01-30T13:44:07.470Z
+guid: 87b1677f-72c8-4b6f-9976-d3cfab075272
+---
+Version control is an essential tool for software developers. It enables tracking changes made to a project's source code, facilitates collaboration among developers, and ensures the project's consistency.
```

Comment:
Hi @PothieuG 

- [ ] 1. This sentence talks about Source Control as a tool... not about tools for Source Control... as the title of the rule implies. Can you improve this intro? 

**TIp:** Use ChatGPT